app/src/main/java/org/oppia/app/player/exploration/ExplorationManagerFragment.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.app.player.exploration
+
+import android.content.Context
+import android.os.Bundle
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/**
+ * FragmentManager of [ExplorationFragment] that observes data provider that retrive default story text size.
```

Comment:
`FragmentManager` in android has a specific meaning and this class is not a `FragmentManager`.
So I think the kdoc can be updated to remove the use of that word.
Something like:
`Fragment manager of [ExplorationFragment] that observes data provider that retrieve default story text size.`

Reply:
Also typo: `retrieve`